modules/api/pkg/ee/clusterbackup/storage-location/handler.go

```diff
@@ -162,6 +171,25 @@ func GetCSBL(ctx context.Context, request interface{}, userInfoGetter provider.U
 	}, nil
 }
 
+func ListCSBLBucketObjects(ctx context.Context, request interface{}, userInfoGetter provider.UserInfoGetter, provider provider.BackupStorageProvider, projectProvider provider.ProjectProvider) (apiv2.BackupStorageLocationBucketObjectList, error) {
+	req, ok := request.(listCbslBucketObjectsReq)
+	if !ok {
+		return nil, utilerrors.NewBadRequest("invalid request")
+	}
+
+	user, err := userInfoGetter(ctx, req.ProjectID)
+
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Please use https://github.com/kubermatic/dashboard/blob/main/modules/api/pkg/ee/clusterbackup/backupstoragelocation/handler.go#L120-L131 to validate the user here